homeassistant/components/plugwise/switch.py

```diff
@@ -30,6 +30,7 @@
         name="Lock",
         icon="mdi:lock",
         entity_category=EntityCategory.CONFIG,
+        entity_registry_enabled_default=False,
```

Comment:
Why is it disabled?

Reply:
Also, less interesting numbers, per 5 mins or per hour, but the graph updates per 5 min. So completely useless.

Reply:
It is not a number but a config entity (switch). It doesn't have graphs...

Reply:
Yeah, the comment was meant for the above chat.

Reply:
Reverted.